sympy/printing/ccode.py

```diff
@@ -298,6 +303,8 @@ def ccode(expr, assign_to=None, **settings):
         Setting contract=False will not generate loops, instead the user is
         responsible to provide values for the indices in the code.
         [default=True].
+    declare: string, optional
+        When given, it specifies the data type of `assign_to` variable.
```

Comment:
The above isn't true for this case: `ccode(a, declare="double")`. Please make it generalize to what it actually does.